python/cudf/cudf/_lib/pylibcudf/libcudf/binaryop.pxd

```diff
@@ -1,9 +1,11 @@
 # Copyright (c) 2020-2024, NVIDIA CORPORATION.
 
-from libc.stdint cimport int32_t
+from libc.stdint cimport bool, int32_t
+from libcpp cimport bool
```

Comment:
Why do we `cimport bool` from two places? This is probably not right.

Reply:
libcpp is the right place.